crates/uv/src/commands/project/run.rs

```diff
@@ -107,11 +107,19 @@ pub(crate) async fn run(
     // Determine whether the command to execute is a PEP 723 script.
     let temp_dir;
     let script_interpreter = if let Some(script) = script {
-        writeln!(
-            printer.stderr(),
-            "Reading inline script metadata from: {}",
-            script.path.user_display().cyan()
-        )?;
+        if let uv_scripts::Source::File(path) = &script.source {
+            writeln!(
+                printer.stderr(),
+                "Reading inline script metadata from: {}",
+                path.user_display().cyan()
+            )?;
+        } else {
+            writeln!(
```

Comment:
Tried to move the `match` within writelin! but couldn't get the arm types to match.